tools/maven/suppressions-java.xml

```diff
@@ -24,6 +24,9 @@ under the License.
 
 <suppressions>
 	<suppress
-		files=".*"
-		checks="NewlineAtEndOfFile|RegexpSingleline|TodoComment|RedundantImport|ImportOrder|RedundantModifier|JavadocMethod|JavadocParagraph|JavadocType|JavadocStyle|PackageName|TypeNameCheck|ConstantNameCheck|StaticVariableNameCheck|MemberNameCheck|MethodNameCheck|ParameterName|LocalFinalVariableName|LocalVariableName|LeftCurly|UpperEll|FallThrough|reliefPattern|SimplifyBooleanExpression|EmptyStatement|ModifierOrder|EmptyLineSeparator|WhitespaceAround|WhitespaceAfter|NoWhitespaceAfter|NoWhitespaceBefore|OperatorWrap|ParenPad"/>
+		files="ExecutionEnvironment\.java"
+		checks="JavadocParagraph"/>
+	<suppress
```

Comment:
In NestedMethodAnalyzer:358-362, there is an intentional fall through which I would require a bit bigger refactoring to eliminate.

For the ExecutionEnvironment it was because we linked in javadoc to class from a package that flink-java does not depend on. I've checked the generated docs, and the links do not work nevertheless, so I removed the suppression and changed `{@link` to `{@code`.